release/ray_release/alerts/handle.py

```diff
@@ -35,6 +43,9 @@ def handle_result(test: Test, result: Result):
     handler = result_to_handle_map[alert_suite]
     error = handler(test, result)
 
+    if error == "Result is empty!":
+        # A proper FetchResultError to be raised outside.
+        return True
```

Comment:
Instead of changing the API here, ca we just raise a `ResultsEmptyError` in the respective alerts?